pkg/audit/audit.go

```diff
@@ -70,3 +71,10 @@ func (a *Audit) Log(name, op, value string) {
 	}
 	a.mutex.Unlock()
 }
+
+func Delete(file string) {
+	check_file, _ := os.Stat(file)
+	if check_file.Size() == 0 && file == "pvsadm_audit.log" {
```

Comment:
why do you wanna check the filename here?

Reply:
This is to delete the file only if its created by the tool and not to delete the user provided audit file.

Reply:
IMO its okay to delete the use provided as well if file is empty

Reply:
ok. will remove the condition and delete the user provided file as well.

Reply:
Done